spec/lib/task-data/schemas/install-coreos-spec.js

```diff
@@ -16,7 +16,8 @@ describe(require('path').basename(__filename), function() {
         "installDisk": "/dev/sda",
         "comport": "ttyS0",
         "osType": "linux",
-        "rootPassword": "RackHD"
+        "rootPassword": "RackHD",
+        "ignitionScriptUri": "http://172.31.128.9090/api/current/templates/ignition.json"
```

Comment:
Thank you for this information. Is this something you'd like to see updated prior to merge?